deploy/converged/olm-catalog/kubevirt-hyperconverged/0.0.1/kubevirt-hyperconverged-operator.v0.0.1.clusterserviceversion.yaml

```diff
@@ -1171,11 +1171,11 @@ spec:
                     valueFrom:
                       fieldRef:
                         fieldPath: metadata.namespace
-                  image: rthallisey/hyperconverged-cluster-operator:latest
+                  image: docker.io/rthallisey/hyperconverged-cluster-operator:latest
                   imagePullPolicy: Always
-                  name: hyperconverged-cluster-operator
+                  name: rthallisey/hyperconverged-cluster-operator
```

Comment:
This change appeared after ./hack/build-manifests.sh. Is that intended @rthallisey ?

Reply:
That doesn't look right.   Something might be wrong in `build-manifests.sh`

Reply:
Should be fixed now